spec/controllers/idv/gpo_verify_controller_spec.rb

```diff
@@ -213,6 +213,10 @@
             )
           end
 
+          before do
+            allow(user).to receive(:fraud_rejection?).and_return(true)
+          end
```

Comment:
Following up on pairing conversation for others reading the PR: We should be able to delete the proofing components from this test setup if we're only looking at profiles. However, in_person_enrollment is still determined from the ProofingComponent, and GpoVerifyForm is looking at ProofingComponents for both in_person_enrollment and threatmetrix status. Changing the code is relatively straightforward. Changing the test setup is much more complex. There's a lot of room for consolidation and refactoring in how we check threatmetrix and in_person_enrollment status throughout the code.